src/components/NavBar/index.tsx

```diff
@@ -64,6 +70,7 @@ function NavBar({ classes }: NavBarProps) {
               className={classes.logo}
               component={Link}
               to="/"
+              onClick={clearAnalysis}
```

Comment:
We really shouldn't need to do this. What if we navigate to `/` somewhere else in the codebase? We would need to include this function everywhere. 

A better way to do this would be to rely on the `useEffect` to clear the pending analysis if it detects the url has changed to `/`. This should happen automatically [here](https://github.com/WFP-VAM/prism-frontend/pull/120/files#diff-314a2832937cda41c6f51176b41683977e412b5dc6b10d8ffff85ffa2f293bd1R204)